### PR TITLE
Apply suggestions from SonarQube

### PR DESCRIPTION
**Pull request by t3chguy**
---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -676,7 +676,13 @@ export function prevString(s: string, alphabet = DEFAULT_ALPHABET): string {
...
     // Dev note: this exists because I'm sad that you can use math operators on strings, so I've
     // hidden the operation in this function.
-    return (a < b) ? -1 : ((a === b) ? 0 : 1);
+    if (a < b) {
+        return -1;
+    } else if (a > b) {
+        return 1;
+    } else {
+        return 0;
+    }
```
Comment (turt2live) [inline src/utils.ts:685]: I mean, sure. The code was clear enough before though I think.

File: src/utils.ts [hunk 2]
```diff
@@ -705,7 +711,6 @@ export function recursivelyAssign(target: Object, source: Object, ignoreNullish
         }
         if ((sourceValue !== null && sourceValue !== undefined) || !ignoreNullish) {
             target[sourceKey] = sourceValue;
-            continue;
```
Comment (turt2live) [inline src/utils.ts:708]: merge blocker: the benefit of being explicit here really helps, I think.

**Review by turt2live — COMMENTED** [review]
only the one actual blocker to merge

File: src/utils.ts (hunk 1 above)
Comment (t3chguy) [reply src/utils.ts:685]: It got upset about nested ternaries, and I don't blame it

**Review by turt2live — APPROVED** [review]

**Pull request merged**